config/server.py

```diff
@@ -264,3 +264,6 @@
                        "processed recursively depth-first, later values "
                        "override earlier ones.")
 
+config_lib.DEFINE_bool("AdminUI.protect_hunt_results", default=True,
```

Comment:
I did this for the logic later of if X and Y for the delete.  I can change that to if not X and Y and make this change if you prefer.